Dockerfile

```diff
@@ -84,11 +84,11 @@ RUN mkdir -p /aws/.terraform.d/plugins && \
         rm -rf /tmp/build \
     ; done
 
-FROM unifio/covalence:0.7.6
+FROM unifio/covalence:0.7.8
 LABEL maintainer="WhistleLabs, Inc. <devops@whistle.com>"
 
 LABEL packer_version="1.0.0"
-LABEL terraform_version="0.10.7"
+LABEL terraform_version="0.10.8"
```

Comment:
I meant to look into this, but forgot.  Do you think there's a way we can combine the LABEL and ENV for the packer and terraform versions so it's only specified in one place?